src/node/desktop/src/main/preferences/electron-desktop-options.ts

```diff
@@ -268,27 +271,42 @@ export class DesktopOptionsImpl implements DesktopOptions {
   }
 
   // Windows-only option
-  public setRBinDir(rBinDir: string): void {
+  public rBinDir(): string {
+    if (process.platform !== 'win32') {
+      return '';
+    }
+
+    let rBinDir: string = dirname(this.rExecutablePath());
```

Comment:
Will this do the right thing if `rExecutablePath()` is not set, or is the empty string?

```
$ node -p 'require("path").dirname("")'
.
```

I think you need to check the value of  `this.rExecutablePath()` before using it.

Reply:
Added this check on [fc3a811](https://github.com/rstudio/rstudio/pull/11168/commits/fc3a8110b86fb07c1bf205513b80fa921b242af7)